mainapp/models.py

```diff
@@ -117,6 +120,8 @@ class Request(models.Model):
         default = 'new'
     )
     supply_details = models.CharField(max_length=100, blank=True)
+    additional_phone_numbers = ArrayField(models.CharField(max_length=200), blank=True, default=list,
```

Comment:
Why `max_length` to 200, We could do with a much smaller number for two phone numbers right ?

Reply:
I did not want to restrict to two phone numbers hence the choice of 200. Can be reduced to 100 which would mean a person can put in 9 phone numbers at the max.